packages/venia-concept/src/components/Checkout/flow.js

```diff
@@ -79,7 +81,13 @@ class Flow extends Component {
             submitShippingMethod
         } = actions;
 
-        const { editing, step, submitting } = checkout;
+        const {
+            editing,
+            step,
+            submitting,
+            isAddressIncorrect,
+            incorrectAddressMessage
```

Comment:
I'll stop commenting on each one, but alphabetical order all around please 😃

Reply:
https://github.com/magento-research/pwa-studio/issues/683. @supernova-at, what do you think about configuring our code quality tools to check this?

Reply:
@supernova-at sorted those ones